README.md

```diff
@@ -188,6 +188,39 @@ in these requests.
 
 See the [FitBit sample](samples/FitBit.gs) for the compelte code.
 
+#### Modifying the access token payload
+Similar to Setting additional token headers, some services, such as the Smartsheet API, require you to [add a hash to the access token request payloads](http://smartsheet-platform.github.io/api-docs/?javascript#oauth-flow). The `setTokenPayloadHandler` method allows you to pass in a function to modify the payload of an access token request before the request is sent to the token endpoint:
+ 
+ 
+    // Set the handler for adding Smartsheet's required SHA hash parameter to the payload:
+    .setTokenPayloadHandler(smartsheetTokenHandler)
+    ...
+    function smartsheetTokenHandler(payload) {
```

Comment:
This sample includes a lot of other logic specific to SmartSheet. Instead of embedding just a snippet of the code, add a full SmartSheet sample to the samples directory, following the pattern of the other samples.